### DISCUSSION TITLE
How do we add a CNAME record to our new subdomain DNS?

### DISCUSSION THREAD

**Discussion by branhoff** [is-a-dev/register#4301] [Q&A] [answered]
I recently merged the `brandon-hoffman.is-a.dev` subdomain. However, when I try to link it to my github page I'm getting the following error:

```
brandon-hoffman.is-a.dev is improperly configured
Your site's DNS settings are using a custom subdomain, brandon-hoffman.is-a.dev, 
that is set up as an A record. 
We recommend you change this to a CNAME record pointing to branhoff.github.io. 
For more information, see documentation (InvalidARecordError).
```

It looks like the domain is registered with NameCheap and I can see they have documentation on how to link the domain to Github Pages: https://www.namecheap.com/support/knowledgebase/article.aspx/9645/2208/how-do-i-link-my-domain-to-github-pages

However, how would I do this as a subdomain?

**Comment by wdhdev** [comment, marked as answer]
@branhoff It seems your domain is setup with a `URL` record which is a redirect. If you want to host your website on that domain change the record to a `CNAME` record pointing to `branhoff.github.io`

**Reply by branhoff** [reply]
Got it. I'll resubmit. Thanks for that.
